### PR TITLE
[JENKINS-65161] fix digester security configuration

### PR DESCRIPTION
**Pull request by olamy**

### CODE REVIEW

**Comment by olamy** [comment]
see https://github.com/jenkinsci/jenkins/pull/5320

**Comment by olamy** [comment]
@candiduslynx ping as you are the main maintainer

File: src/main/java/org/jenkinsci/plugins/IBM_zOS_Connector/SCLMChangeLogParser.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import org.apache.commons.digester3.Digester;
 import org.xml.sax.SAXException;
 
+import javax.xml.parsers.ParserConfigurationException;
```
Comment (rsandell) [inline src/main/java/org/jenkinsci/plugins/IBM_zOS_Connector/SCLMChangeLogParser.java:10]: not `digerster3`?

**Review by rsandell — APPROVED** [review]

**Review by alecharp — APPROVED** [review]

File: src/main/java/org/jenkinsci/plugins/IBM_zOS_Connector/SCLMChangeLogParser.java (hunk 1 above)
Comment (olamy) [reply src/main/java/org/jenkinsci/plugins/IBM_zOS_Connector/SCLMChangeLogParser.java:10]: line 7 ;) it was there already

**Review by MRamonLeon — APPROVED** [review]

**Review by candiduslynx — APPROVED** [review]

**Comment by olamy** [comment]
@candiduslynx thanks a lot for the merge and the release!

**Pull request merged**